### PR TITLE
inline raise and others

### PR DESCRIPTION
**Pull request by dsyme**
Address suggestion https://github.com/Microsoft/visualfsharp/issues/3121

It is correct that the debugging experience is nicer if this is done.  I've noticed this several times recently

We may as well mark a couple of other simple functions as "inline" too

### CODE REVIEW

File: src/fsharp/FSharp.Core/prim-types.fsi [hunk 1]
```diff
@@ -2189,7 +2189,7 @@ namespace Microsoft.FSharp.Core
         /// <param name="x">The input value.</param>
         /// <returns>The same value.</returns>
         [<CompiledName("Identity")>]
-        val id : x:'T -> 'T 
+        val inline id : x:'T -> 'T
```
Comment (forki) [inline src/fsharp/FSharp.Core/prim-types.fsi]: that one is important ;-)
But would it be automatically inlined because of beta reduction?

Comment (dsyme) [inline src/fsharp/FSharp.Core/prim-types.fsi]: In optimized code all these are always inlined.

**Comment by dsyme** [comment]
Failing codegen tests. Serves me right :)

```
CodeGen\EmittedIL\ComputationExpressions (ComputationExpr04.fs) -- failed
CodeGen\EmittedIL\QueryExpressionStepping (Linq101Aggregates01.fs - CodeGen) -- failed
CodeGen\EmittedIL\SeqExpressionStepping (SeqExpressionSteppingTest7.fs - NetFx40) -- failed
CodeGen\EmittedIL\TestFunctions (TestFunction3b.fs - NetFx40) -- failed
CodeGen\EmittedIL\TestFunctions (TestFunction3c.fs - NetFx40) -- failed
```

**Comment by dsyme** [comment]
OK, this is ready

**Pull request merged**